src/model/index.py

```diff
@@ -0,0 +1,18 @@
+import os
+from output.exceptions import *
+
+
+class Index:
+    def __init__(self, names, columns):
+        self.names = names
+        self.validate(self, columns)
```

Comment:
Validate is a static private method.
Because it is a static, it shouldn’t take self or shouldn’t be called `self.`

it should be private because it is called by constructor only. It should start with `__`

Reply:
It throws an error when not get called self. and when the parameter self isn't given